showyourwork/workflow/rules/compile.smk

```diff
@@ -62,10 +65,13 @@ rule:
             --chatter minimal     \\
             --keep-logs           \\
             --keep-intermediates  \\
-            --synctex             \\
-            "{input[0]}"
+            {maybe_synctex}       \\
+            "{input[0]}" {joined_user_args}
```

Comment:
Then here I don't think this will work as currently written because the rule won't have access to the globals. Instead, this should be:

```suggestion
            {params.maybe_synctex}   \\
            {params.user_args}             \\
            "{input[0]}"
```